launcher/Application.cpp

```diff
@@ -415,11 +419,11 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
                 QString(
                     "The launcher couldn't create a log file - the data folder is not writable.\n"
                     "\n"
-                    "Make sure you have write permissions to the data folder.\n"
+                    "Make sure you have write permissions to the logs folder.\n"
                     "(%1)\n"
                     "\n"
                     "The launcher cannot continue until you fix this problem."
-                ).arg(dataPath)
+                ).arg(dataPath+"/logs")
```

Comment:
I think this should keep saying data folder. Because the launcher would fail later as well.